### DISCUSSION TITLE
Understanding Hierarchical Levels in Community Detection with the Leiden Algorithm

### DISCUSSION THREAD

**Discussion by sakthii25** [microsoft/graphrag#1128] [Algorithm + Paper]
In the Leiden algorithm for community detection, the process involves multiple hierarchical levels:

Level 0: Start with the original graph. Communities are detected through local moving and refinement techniques.
Level 1: Aggregate the detected communities into a new graph. Apply the Leiden technique again to find larger communities.
Higher Levels: Continue aggregating and applying the technique until no further improvement in community structure can be achieved.
As you progress to higher levels, the number of communities typically decreases. This hierarchical approach helps in refining community structures and identifying larger, more meaningful communities at each level.

**Is my understanding correct?** If there are any nuances or additional details to consider, please let me know.

**Comment by hhh2210** [comment]
I think your understanding is fabulus! 
THe official report dont reveal the important details
